renovate.json

```diff
@@ -12,31 +12,42 @@
   "prHourlyLimit": 0,
   "schedule": ["after 7am and before 9am every weekday"],
   "separateMajorMinor": false,
-  "regexManagers":[
+  "regexManagers": [
     {
       "depNameTemplate": "defenseunicorns/uds-core",
       "fileMatch": ["README\\.md"],
-      "matchStrings": [
-        "- UDS Core: (?<currentValue>.*)"
-      ],
+      "matchStrings": ["- UDS Core: (?<currentValue>.*)"],
       "datasourceTemplate": "github-tags",
       "extractVersionTemplate": "^v(?<version>.*)$"
     },
     {
       "depNameTemplate": "defenseunicorns/uds-cli",
       "fileMatch": ["README\\.md"],
-      "matchStrings": [
-        "- UDS CLI: (?<currentValue>.*)"
-      ],
+      "matchStrings": ["- UDS CLI: (?<currentValue>.*)"],
       "datasourceTemplate": "github-tags",
       "extractVersionTemplate": "^v(?<version>.*)$"
     },
     {
       "depNameTemplate": "k3d-io/k3d",
       "fileMatch": ["README\\.md"],
+      "matchStrings": ["- K3D: (?<currentValue>.*)"],
+      "datasourceTemplate": "github-tags",
+      "extractVersionTemplate": "^v(?<version>.*)$"
+    },
+    {
+      "depNameTemplate": "defenseunicorns/lula",
+      "fileMatch": ["README\\.md"],
+      "matchStrings": ["- Lula: (?<currentValue>.*)"],
+      "datasourceTemplate": "github-tags",
+      "extractVersionTemplate": "^v(?<version>.*)$"
+    },
+    {
+      "fileMatch": [".github/actions/setup/action\\.yaml"],
       "matchStrings": [
-        "- K3D: (?<currentValue>.*)"
+        "uses: defenseunicorns/lula-action/setup@(?<currentValue>[a-f0-9]{40})",
+        "version: v(?<currentValue>\\d+\\.\\d+\\.\\d+)"
       ],
+      "depNameTemplate": "defenseunicorns/lula",
       "datasourceTemplate": "github-tags",
       "extractVersionTemplate": "^v(?<version>.*)$"
```

Comment:
This is not needed (and I am not sure it would work as expected? - usually for new renovate adds that aren't simple copies I'd ask for a test) - I made another suggestion above that would use the common renovate config.

Reply:
Ahh I needed to do the above - I am a renovate noob.